recipes/mplcursors/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "mplcursors" %}
+{% set version = "0.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/m/mplcursors/mplcursors-{{ version }}.tar.gz
+  sha256: 67fbbc5c7ea979c223baf01df55e3cab425f71d3fb96ed100686873ddf3ae786
+
+build:
+  number: 0
+  noarch: python
+  script:
+    # include the tests in the conda distribution
+    - mv tests mplcursors/.
```

Comment:
I need to move the tests in manually. Is this preferred?

Reply:
You typically don't need to run the full test suite. We're mostly looking for existence tests and that everything is imported correctly.

Reply:
https://conda-forge.org/docs/testing.html#simple-existence-tests

Reply:
Got it thanks. Hopefully it builds now